pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -185,7 +185,7 @@ let
       owner = "tensorflow";
       repo = "tensorflow";
       rev = "v${version}";
-      sha256 = "sha256-n7jRDPeXsyq4pEWSWmOCas4c8VsArIKlCuwvSU/Ro/c=";
+      sha256 = "1qwzbqq899swrwrwmm6z7mq9sc55gyh0r4ca0mcnchbvn7w0qbkh";
```

Comment:
You need to also update `fetchAttrs.sha256`

Reply:
I tried but it resulted in the same hash for cuda and non-cuda variants.

Reply:
You are right. It did not occur to me that the hash may not change in a minor update (it did in the past). Thank you for checking. I did just check that the `python3Packages.tensorflow.deps` builds on darwin.